cookbooks/aws-parallelcluster-install/recipes/install_mysql_client.rb

```diff
@@ -24,7 +24,7 @@
 
   package_installer = value_for_platform(
     'default' => "yum install -y",
-    'ubuntu' => { 'default' => "apt install -y" }
+    'ubuntu' => { 'default' => "apt install -y --allow-downgrades" }
```

Comment:
We're not sure we want to include this change, this is the reason we didnt' merge yet the patch: https://github.com/aws/aws-parallelcluster-cookbook/pull/1680

Reply:
Removing the commit.